api/desecapi/models.py

```diff
@@ -366,7 +366,6 @@ def save(self, *args, **kwargs):
         self.iban = self.iban[:6] + "xxx" # do NOT save account details
         super().save(*args, **kwargs) # Call the "real" save() method.
 
-
```

Comment:
PEP 8 states:
> Surround top-level function and class definitions with two blank lines

While not my style here, the PEP text is actually very clear. But let's not have this hold up the PR.